fidesctl/src/fidesctl/cli/cli.py

```diff
@@ -316,4 +321,11 @@ def webserver(ctx: click.Context) -> None:
     """
     Starts the fidesctl API server.
     """
+    try:
+        import fastapi
+    except ModuleNotFoundError as err:
+        echo_red('FastAPI not found. Try: pip install "fidesctl[webserver]"')
+        raise SystemExit
+    from fidesapi.main import start_webserver
+
```

Comment:
We should add a similar block that checks for the DB dependencies within the `init_db` and `reset_db` commands, given that `postgres` and `mysql` are "extras" now as well, no?

Reply:
good catch. that's going to get torn out in #202 , so we'd just be adding it in to immediately take it out

if you feel strongly about it though then i can go ahead and add it

Reply:
Nope, looks like #202 is prioritized so no sense in adding the code.